dockerfiles/Dockerfile.router

```diff
@@ -1,39 +1,43 @@
 ARG DEBUG_IMAGE
 # Build is required to extract the release files
-FROM --platform=linux/amd64 alpine:latest AS build
+FROM --platform=linux/amd64 debian:bullseye-slim AS build
 
-ARG ROUTER_RELEASE
+ARG ROUTER_RELEASE=latest
 
-# Pull release from GH
-ADD https://github.com/apollographql/router/releases/download/v${ROUTER_RELEASE}/router-${ROUTER_RELEASE}-x86_64-unknown-linux-gnu.tar.gz /tmp/router.tar.gz
+WORKDIR /dist
 
-WORKDIR /tmp
```

Comment:
We no longer need to work in a `/tmp` directory and then moving it to a `/dist` since the installer actually just takes care of the extraction itself.